sdk/tests/conformance/textures/misc/tex-image-svg-image-no-natural-width-and-height.html

```diff
@@ -0,0 +1,65 @@
+<!--
+Copyright (c) 2025 The Khronos Group Inc.
+Use of this source code is governed by an MIT-style license that can be
+found in the LICENSE.txt file.
+-->
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>WebGL texImage2D w/ &lt;img> with SVG image without natural width and height</title>
+<link rel="stylesheet" href="../../../resources/js-test-style.css"/>
+<script src="../../../js/js-test-pre.js"></script>
+<script src="../../../js/webgl-test-utils.js"> </script>
+<div id="description"></div>
+<div id="console"></div>
+<script>
+"use strict";
+
+description("Test texImage2D from an img element with an SVG image without natural width and height.");
+
+const wtu = WebGLTestUtils;
+
+function test(image, variant) {
+  debug('');
+  debug(variant);
+
+  const tex = gl.createTexture();
+  gl.bindTexture(gl.TEXTURE_2D, tex);
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_S, gl.CLAMP_TO_EDGE);
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_T, gl.CLAMP_TO_EDGE);
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.NEAREST);
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.NEAREST);
+
+  if (variant === "texSubImage2D") {
+    gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, image.width, image.height, 0, gl.RGBA,
+                  gl.UNSIGNED_BYTE, null);
+    gl.texSubImage2D(gl.TEXTURE_2D, 0, 0, 0, gl.RGBA, gl.UNSIGNED_BYTE, image);
+  } else {
+    gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, gl.RGBA, gl.UNSIGNED_BYTE, image);
+  }
+
+  wtu.clearAndDrawUnitQuad(gl);
+
+  wtu.checkCanvasRect(gl, 75, 37, 2, 2, [0, 0, 255, 255], "should be blue");
+  wtu.checkCanvasRect(gl, 75, 108, 2, 2, [0, 0, 0, 0], "should be transparent");
+  wtu.checkCanvasRect(gl, 225, 108, 2, 2, [255, 255, 0, 255], "should be yellow");
+  wtu.checkCanvasRect(gl, 225, 37, 2, 2, [0, 0, 0, 0], "should be transparent");
+}
+
+const SVG_IMAGE = `<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 100 100'>
+                     <rect width='50' height='50' fill='blue'/>
+                     <rect x='50' y='50' width='50' height='50' fill='yellow'/>
+                   </svg>`;
+const gl = wtu.create3DContext();
+const program = wtu.setupTexturedQuad(gl);
+
+wtu.loadImageAsync(`data:image/svg+xml,${SVG_IMAGE}`, image => {
+  image.width = 100;
+  image.height = 100;
```

Comment:
Also perhaps due to the ambiguity above we need a test case that doesn't have width/height on the img?